e2e/interchaintestv8/e2esuite/utils.go

```diff
@@ -56,17 +56,19 @@ import (
 // BroadcastMessages broadcasts the provided messages to the given chain and signs them on behalf of the provided user.
 // Once the broadcast response is returned, we wait for two blocks to be created on chain.
 func (s *TestSuite) BroadcastMessages(ctx context.Context, chain *cosmos.CosmosChain, user ibc.Wallet, gas uint64, msgs ...sdk.Msg) (*sdk.TxResponse, error) {
+	sdk.GetConfig().SetBech32PrefixForAccount(chain.Config().Bech32Prefix, chain.Config().Bech32Prefix+sdk.PrefixPublic)
+	sdk.GetConfig().SetBech32PrefixForValidator(
+		chain.Config().Bech32Prefix+sdk.PrefixValidator+sdk.PrefixOperator,
+		chain.Config().Bech32Prefix+sdk.PrefixValidator+sdk.PrefixOperator+sdk.PrefixPublic,
+	)
+
 	broadcaster := cosmos.NewBroadcaster(s.T(), chain)
 
 	broadcaster.ConfigureClientContextOptions(func(clientContext client.Context) client.Context {
-		txConfig := chain.Config().EncodingConfig.TxConfig
-		return clientContext.WithCodec(chain.Config().EncodingConfig.Codec).
-			WithTxConfig(txConfig).
-			WithAddressCodec(txConfig.SigningContext().AddressCodec()).
-			WithValidatorAddressCodec(txConfig.SigningContext().ValidatorAddressCodec()).
-			WithAddressPrefix(chain.Config().Bech32Prefix).
-			WithAddressCodec(txConfig.SigningContext().AddressCodec()).
-			WithValidatorAddressCodec(txConfig.SigningContext().ValidatorAddressCodec())
+		return clientContext.
+			WithCodec(chain.Config().EncodingConfig.Codec).
+			WithChainID(chain.Config().ChainID).
+			WithTxConfig(chain.Config().EncodingConfig.TxConfig)
```

Comment:
I think some of these changes shouldn't be reverted if possible. (May not be possible)

Reply:
The things removed here were just 0.52 stuff